### ISSUE TITLE
[check package] Constructor property in an object

### ISSUE THREAD

**Issue by jagi** [meteor/meteor#5965]
Hi, there is probably a bug in the Match.test function when checking the `"constructor"` property value. The following code will return false:

``` js
var o = {
  constructor: 'abc'
};
Match.test(o, {constructor: String}); // Returns false
```

which of course is wrong.

**Comment by mutdmour** [comment]
cannot replicate this in `devel`

**Comment by jagi** [comment]
It was probably fixed in some of the lately released versions. I don't remember in what version was this bug present but it was confirmed. I've checked it one more time now and it works, so we can close this issue.

**Comment by abernix** [comment]
This was likely fixed by the same fix which resolved #7354 which was included in PR https://github.com/meteor/meteor/pull/8351 by @hwillson and fixed in `check@1.2.5`.

Thanks for pointing this out, @mutdmour.  And thanks for following-up with the close, @jagi.
